plugin/cmd/kube-scheduler/app/server.go

```diff
@@ -77,6 +77,9 @@ func Run(s *options.SchedulerServer) error {
 	// cache only non-terminal pods
 	podInformer := factory.NewPodInformer(kubecli, 0)
 
+	// Install algorithm provider and algorithms before scheduling.
+	algorithmprovider.Install()
```

Comment:
I believe the same needs to be done in `./app/configurator.go`. Maybe... This change can actually break some tooling, but I'm not sure how much we care. @kubernetes/sig-scheduling-pr-reviews